desktop/npm-helper.js

```diff
@@ -179,6 +179,14 @@ const commands = {
     help: 'Undiff log send',
     code: undiff,
   },
+  'generate-font-project': {
```

Comment:
yeah. i tried to avoid icomoon but its too much work to fit into this ticket. the good news is that we can rewrite the generator to whatever new format (vs doing it by hand like before)